postfix/ci/postfix.rake

```diff
@@ -0,0 +1,64 @@
+require 'ci/common'
+
+def postfix_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+def postfix_rootdir
```

Comment:
We don't need this anymore, you can remove it. We should also remove it from the gem, it puts it everywhere. No harm done, but useless.